demos/vite-demo/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "vite-demo",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview",
+    "iso": "iso --watch"
+  },
+  "dependencies": {
+    "@isograph/react": "^0.2.0",
```

Comment:
let's install these the same way that they're installed in the other demos, that way we can make local changes and immediately see them reflected here. So:

* in the root package.json, add equivalent `watch-vite-demo` and `build-vite-demo` and adjust the other commands
* add stuff to `ci.yml`, namely ensuring that when we run `build-vite-demo` the working directory is clean
* here, `"@isograph/react": "*",` and installed via pnpm

Reply:
Done!